src/sbt-test/sbt-plugins/simple/core/src/test/scala/SharedTestHelper.scala

```diff
@@ -1,5 +1,5 @@
 package test
 
 object SharedTestHelper {
-  def sayHi: String = "hi"
+  def sayHi(): String = "hi"
```

Comment:
Really? That's super strange - you're not supposed to use parens in no-arg functions that don't have side-effects . . .